www/templates/home-epaper.html

```diff
@@ -0,0 +1,23 @@
+<ion-view title="诗华电子报">
+    <ion-nav-bar class="bar-assertive">
+        <ion-nav-buttons side="left">
+            <!-- Toggle left side menu -->
+            <button menu-toggle="left" class="button button-icon icon ion-navicon"></button>
+        </ion-nav-buttons>
+        <ion-nav-buttons side="right">
+            <a href="#/app/breakingnewslist" class="button" style="right:25px">
+                即时新闻
+            </a>
+            <span class="badge badge-light" style="position: absolute; right: 0px" ng-show="breakingNewsCount > 0">!</span>
+            
+        </ion-nav-buttons>
+    </ion-nav-bar>
+    <ion-content ng-controller="MenuCtrl" style="text-align: center; background-color: #FF0000;">
+	<h3 class="title" style="text-align: center; background-color: #FF0000; color: #FFFFFF;">See Hua e-Paper</h3>
```

Comment:
Title should be in Chinese. As of now, we can use English title first. I will create another JIRA ticket for this.